composer.json

```diff
@@ -25,7 +25,7 @@
     "require-dev": {
         "mockery/mockery": "^1.4.2",
         "phpunit/phpunit": "^9.0|^10",
-        "orchestra/testbench": "dev-phpunit10"
+        "orchestra/testbench": "^5.17.1|^6.12.1|dev-phpunit10"
```

Comment:
This is unneeded I think? Only Laravel 8 is supported here.

Reply:
Fixed